pepper-apis/dss-database/src/main/resources/db-changes/schema/DDP-8000.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                   https://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+    <changeSet id="DROP_AUTH0_DOMAIN_COLUMN_AGAIN" author="Dmitrii">
+        <preConditions onError="MARK_RAN" onFail="MARK_RAN">
```

Comment:
Change set will be applied only in the case client.auth0_domain exists. Otherwise, the whole change set will be marked as executed.